README.md

```diff
@@ -1467,3 +1476,17 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents:
```

Comment:
While it's useful to point out README sections that are especially important, it should be made clear that a noob is expected to read the entire README. If you think there are sections that serve no purpose to a noob (I can only think of one), we could point out at the beginning of those sections that a noob can skip them initially.